R/tree_accessors.R

```diff
@@ -2558,11 +2558,12 @@ ct_recursive_replace <- function(ctree, path, value, pos = 1) {
 #' Set visibility of column counts for a group of sibling facets
 #'
 #' @inheritParams gen_args
-#' @param path character. The path *to the parent of the
-#' desired siblings*. The last element in the path should
-#' be a split name.
-#' @return obj, modified with the desired columncount
-#' display behavior
+#' @param path (`character`)\cr the path *to the parent of the
```

Comment:
I know it is boring, but we should keep this style ^^ could you check for the other new parameters?